scripts/consent_form_json.py

```diff
@@ -73,24 +48,23 @@ def get_credentials():
     return credentials
 
 
-def main():
-    credentials = get_credentials()
-    http = credentials.authorize(httplib2.Http())
-    service = discovery.build('drive', 'v3', http=http)
-
-    # get and download files
-    data = service.files().list(q="'" + FOLDER_ID + "'" + " in parents").execute()
+def run(folderid,service):
+    #get and download files
+    data = service.files().list(q="'" + folderid + "'" + " in parents").execute()
     download_files(data['files'], service)
 
     # convert csv files to a single json file
     content = {}
     for filename in os.listdir(files_path):
-        site_id = os.path.splitext(filename)[0]
-        filepath = 'consent_forms/' + filename
-        if not os.path.isfile(filepath):
+        if filename.endswith('.csv'):
+            site_id = os.path.splitext(filename)[0]
+            filepath = 'consent_forms/' + filename
```

Comment:
Ok in this case because this is a very specific script, but in the future, consider using `os.path.join`, which is more platform agnostic.

For example, on windows, folder names are separated by a `\`. Using the convenience methods provided by python can help a script run better on different operating systems.